jaxlib/kernel_helpers_no_pybind11.h

```diff
@@ -0,0 +1,48 @@
+/* Copyright 2019 Google LLC
```

Comment:
Rather than name files based on what they aren't, I'd rather name them based on what they are (perhaps: `kernel_helpers.h` and `kernel_pybind_helpers.h`?)

Reply:
I deliberated over this for a while and eventually concluded that no_pybind11 is the exceptional case so should have the qualified name. I was on the fence though, so changed to kernel_helpers.h and kernel_pybind11_helpers.h (pybind11 so it's consistent everywhere).